packages/admin/admin/src/translator/config/TranslationConfigContext.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from "react";
+
+export interface TranslationConfig {
+    enabled: boolean;
```

Comment:
I don't see a point in this flag, better make the whole context optional (`React.createContext<TranslationConfig | undefined>` with default value of `undefined`)

Reply:
I believe the idea behind this flag is to be able to disable the service based on the current scope. How would you achieve this instead? Conditionally render the provider?

Reply:
Yes its for disabling it based on the current scope. The only way to omit the enabled variable, would be making translate option and check if a translated function is set. But think it is better to understand in the project setup with the enabled variable.

```
<ContentTranslationServiceProvider
    enabled={enabledScope[scope]}
    translate={async function (input: string): Promise<string> {
        return yourTranslationFnc(value);
    }}
>
```